src/CompressorPresets.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <array>
+
+struct CompressorPreset {
+  float ratio;
+  float thresholdDB;
+  float attackMS;
+  float releaseMS;
+  float makeUpGainDB;
+  CompressorPreset(float r, float t, float a, float rel, float m)
+    : ratio(r)
+    , thresholdDB(t)
+    , attackMS(a)
+    , releaseMS(rel)
+    , makeUpGainDB(m)
+  {}
+  ~CompressorPreset() = default;
+};
+
+namespace CompressorPresets
+{
+  //                     name   ratio  thresh  attack  rel  mugain
+  const CompressorPreset voice { 2.0f, -24.0f, 15.0f, 40.0f, 2.0f };
+  const CompressorPreset horns { 3.0f, -10.0f, 100.0f, 250.0f, 2.0f };
+  const CompressorPreset snare { 5.0f, -4.0f, 5.0f, 150.0f, 3.0f };
+  const uint numPresets { 3 };
```

Comment:
what about all the others? ;-)

Reply:
I was thinking to be conservative until I have someone who can test and vouch for each particular preset choice, since it will not be good form to change a standard preset later after it is in a release.  These presets were obtained from a quick Internet search, and I don't know how good they are.  The two new ones span a large range of compression, and hopefully cover Russ's needs for the Band.